mllib/src/main/scala/org/apache/spark/mllib/tree/RandomForest.scala

```diff
@@ -23,6 +23,7 @@ import scala.util.Try
 import org.apache.spark.annotation.Since
 import org.apache.spark.api.java.JavaRDD
 import org.apache.spark.internal.Logging
+import org.apache.spark.ml.feature.{Instance => NewInstance}
```

Comment:
the mllib code is referencing ml code... scary and weird, but I guess not much you can do here, just seems really convoluted